plugins/algoltek-aux/fu-algoltek-aux-device.h

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright (C) 2024 Algoltek <Algoltek, Inc.>
+ *
+ * SPDX-License-Identifier: LGPL-2.1+
+ */
+
+#pragma once
+
+#include <fwupdplugin.h>
+
+#define FU_TYPE_ALGOLTEK_AUX_DEVICE (fu_algoltek_aux_device_get_type())
+G_DECLARE_FINAL_TYPE(FuAlgoltekAuxDevice,
+		     fu_algoltek_aux_device,
+		     FU,
+		     ALGOLTEK_AUX_DEVICE,
+		     FuDpauxDevice)
+
+FuAlgoltekAuxDevice *
+fu_algoltek_aux_device_new(FuDpauxDevice *device);
```

Comment:
you need a newline at the end of each file otherwise github gets upset

Reply:
OK! I have got it.